docs/docs/general/03-Element Selection/recommended-practices.md

```diff
@@ -174,72 +197,102 @@ await aui.click().button().withText('Sign in');
                  .exec();
 ```
 
-### Icon Detection Fails or Labels Wrong Icon Name
-Detecting icons correctly is difficult because there are millions of icons available. You typically run into two problems:
+### Icon Detection Fails
+Detecting icons correctly is difficult because there are millions of icons available. You run into the problem that an icon is not detected sometimes:
```

Comment:
```suggestion
Detecting icons correctly is difficult because there are millions of icons available. Therefore, you may run into the problem that an icon is not detected sometimes:
```